packages/bierzo-wallet/src/routes/transactions/components/TxTable/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { makeStyles, Theme } from '@material-ui/core';
+import { useTheme } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import * as React from 'react';
+
+import { ProcessedTx } from '../../../../store/notifications';
+import { getShadowColor } from '../../../../theme/css';
+import { CSV_PADDING } from '../DownloadCSV';
+import { TxTableProps } from './rowTxBuilder';
+import TxTableFooter from './TxTableFooter';
+import TxTableHeader from './TxTableHeader';
+import TxTableRow from './TxTableRow';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  inner: {
+    display: 'flex',
+    flex: '1 0 auto',
+    flexDirection: 'column',
+  },
+  outer: {
+    display: 'flex',
+  },
+  panel: {
+    display: 'flex',
+    flexDirection: 'column',
+    backgroundColor: 'white',
+    borderRadius: 4,
+    boxShadow: `0 0 20px 0 ${getShadowColor()}`,
+  },
+  column: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+}));
```

Comment:
Some of the styles above can be moved to `Block` but not all like `borderRadius`, `boxShadow`.